test_project/config/settings_interactive.py

```diff
@@ -0,0 +1,14 @@
+# Normally test_project is used as scaffolding for
+# django_sql_dashboard's automated tests. However, it can
+# be useful during development to have a sample project to
+# tinker with interactively. These Django settings can be
+# useful when we want to do that.
+
+from .settings import *
+
+# Just have our dashboard use the exact same credentials for
+# our database, there's no need to bother with read-only
+# permissions when using test_project interactively.
+DATABASES["dashboard"] = DATABASES["default"]
+
+LOGIN_URL="/admin/login/"
```

Comment:
@simonw strictly speaking, I don't think this necessarily needs to be part of `settings_interactive.py`--it could just be in `settings.py`, but I felt weird modifying any existing files.  If you want, I can move it into `settings.py` though, let me know!

Reply:
Modifying settings.py is fine by me - the only purpose of that module is to get the tests to run.